deployment/v3/mosip/masterdata-loader/install.sh

```
@@ -34,12 +148,13 @@ if [ $yn = "Y" ]
 
    echo Loading masterdata
    helm -n $NS install masterdata-loader mosip/masterdata-loader \
-      --set mosipDataGithubBranch="v1.3.0" \
+      --set mosipDataGithubBranch="release-1.3.x" \
       --set mosipDataGithubRepo="https://github.com/mosip/mosip-data" \
       --set mosipDataXlsfolderPath="\/home/mosip/mosip-data/mosip_master/xlsx" \
+      -f values.yaml \
       --version $CHART_VERSION --wait --wait-for-jobs
 
    else
    break
```

Shellcheck (0.11.0)

[error] break is only valid in loops.

(SC2105)